dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -1133,15 +1141,18 @@ private void initComplementDataParam(ProcessDefinition processDefinition,
             return;
         }
 
-        Date start = DateUtils.stringToDate(cmdParam.get(CMDPARAM_COMPLEMENT_DATA_START_DATE));
-        Date end = DateUtils.stringToDate(cmdParam.get(CMDPARAM_COMPLEMENT_DATA_END_DATE));
+        Date start = null, end = null;
+        if (cmdParam != null) {
+            start = DateUtils.stringToDate(cmdParam.get(CMDPARAM_COMPLEMENT_DATA_START_DATE));
```

Comment:
It would be better to add a `CMDPARAM_COMPLEMENT_DATA_START_DATE` key existence judgment.

Reply:
may be no need